### DISCUSSION TITLE
Get visits of a model between two timeframes

### DISCUSSION THREAD

**Discussion by ataldev** [coderflexx/laravisit#38] [Ideas]
It would be great if we could get the visits a specific model got, between two dates.
Something like:
`$post->visitsThisMonth;
or
$post->visitsThisWeek;
or
$post->visitsToday;
`

**Comment by ousid** [comment]
Hey @ataldev 

Sounds Great! Feel free to create a PR for these methods, with related tests, and I'll try to merge them as soon as I can, once I review.
